docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx

```diff
@@ -139,13 +139,20 @@ Joining an audio room requires microphone access, you need to request permission
 Create `AudioRooms/ContentViewController.swift` and replace it with this code:
 
 ```swift title="AudioRooms/ContentViewController.swift"
+import UIKit
+import Combine
+import StreamVideo
+
 final class ContentViewController: UIViewController {
     private let call: Call
     private var callCreated: Bool = false {
         didSet {
-            var content = contentView.content
-            content.callCreated = true
-            contentView.content = content
+            contentView.content = .init(
+                callCreated: true,
+                callId: call.callId,
+                title: "",
```

Comment:
@ipavlidakis, what title should be used here?

Reply:
I think empty is fine. This title will be later replaced from the descriptionView which will use a Combine publisher to access the title and the description.